yarn-project/end-to-end/src/e2e_nft.test.ts

```diff
@@ -62,21 +62,40 @@ describe('NFT', () => {
     // the sender would be the AMM contract.
     const recipient = user2Wallet.getAddress();
 
```

Comment:
In this file the changes are related to me disabling the "overwriting of transient storage with zeros" change.